test/remote/gateways/remote_paymill_test.rb

```diff
@@ -2,71 +2,79 @@
 
 class RemotePaymillTest < Test::Unit::TestCase
   def setup
-    @gateway = PaymillGateway.new(fixtures(:paymill))
+    @gateway = PaymillGateway.new( public_key: ENV['PAYMILL_API_TEST_PUBLIC_KEY'], private_key: ENV['PAYMILL_API_TEST_PRIVATE_KEY'] )
```

Comment:
Please don't change this not to load from fixtures - this will break the tests for anyone who is running them with paymill credentials in a fixtures file right now